gradle/buildSrc/src/main/kotlin/io/spine/gradle/internal/deps.kt

```diff
@@ -74,42 +74,30 @@ object Repos {
     val gradlePlugins = "https://plugins.gradle.org/m2/"
 }
 
+/**
+ * Versions of one-line dependencies.
+ *
+ * For versions of other dependencies please see `version` properties of objects declared below.
+ */
 object Versions {
-    val checkerFramework = "3.7.1"
-    val errorProne       = "2.4.0"
-    val errorProneJavac  = "9+181-r4173-1" // taken from here: https://github.com/tbroyer/gradle-errorprone-plugin/blob/v0.8/build.gradle.kts
-    val errorPronePlugin = "1.3.0"
-    val pmd              = "6.24.0"
+    val animalSniffer    = "1.19"
```

Comment:
It is a bit odd that some versions are here while others are grouped with the respective artefact classifiers. I guess, we could have a separate object for each dependency group.